File: week-6/series_sum.rb
def series_sum max 
    base_num = 1
    denominator = 4 
    increment = 3 
    max.times do 
        if max == 0
            0
        else if max == 1
            base_num += 1 / denominator
        else 
            base_num += 1 / denominator += increment
        end 
    end 
    p base_num
end 

series_sum 0
series_sum 1
series_sum 2
series_sum 5

File: week-6/seriesSum.js
const seriesSum = (max) => {
    var baseNum = 1
    var denominator = 4
    var incrementor = 3
    for (let i = 0; i <= max; i++) {
        if (max = 0) {
            return 0
        } else if (max = 1) {
            baseNum += (1 / denominator)
        } else {
            baseNum += (1 / (denominator += incrementor))
        }
    }
    return baseNum
}

console.log(seriesSum(0));
console.log(seriesSum(1));
console.log(seriesSum(3));
console.log(seriesSum(5));